Source/Components/ImageGlass.Settings/WebUI/FrmSettings.html

```diff
@@ -743,6 +743,10 @@
           <div class="mb-2">
             <b lang-text="FrmSettings._ImageInterpolation">[Image interpolation]</b>
           </div>
+          <div class="mb-3">
+            <a class="mb-1" lang-text="FrmSettings._ImageInterpolation._MoreInfo" lang-href="FrmSettings._ImageInterpolation._MoreInfoLink">
+              [ImageInterpolationMoreInfo]</a>
+          </div>
```

Comment:
Thanks @stevencaiOR for the PR!
- You can use the language key `_._LearnMore` to show "Learn more…" text instead of "More Info" which introduces new a translation
- About the link, you can use https://learn.microsoft.com/windows/win32/api/d2d1_1/ne-d2d1_1-d2d1_interpolation_mode#constants which auto-redirects to the local language URL instead of passing the localized link to every language file

The HTML for "Learn more" link will just be like this:
```html
<a class="mb-1"
  href="https://learn.microsoft.com/windows/win32/api/d2d1_1/ne-d2d1_1-d2d1_interpolation_mode#constants"
  target="_blank"
  rel="noopener noreferrer"
  lang-text="_._LearnMore">
  [Learn more]
</a>
```

We don't need JavaScript, either.

Reply:
Okay, I made the necessary changes.